backend/ibutsu_server/widgets/compare_runs_view.py

```diff
@@ -0,0 +1,189 @@
+from unittest import result
+import connexion
+import time
+from datetime import datetime
+from datetime import timedelta
+
+from ibutsu_server.constants import JJV_RUN_LIMIT
+from ibutsu_server.db.base import Integer
+from ibutsu_server.db.base import session
+from ibutsu_server.db.base import Text
+from ibutsu_server.db.models import Run
+from ibutsu_server.db.models import Result
+from ibutsu_server.filters import apply_filters
+from ibutsu_server.filters import convert_filter
+from ibutsu_server.filters import string_to_column
+from ibutsu_server.util.count import get_count_estimate
+from ibutsu_server.util.projects import add_user_filter
+from sqlalchemy import desc
+from sqlalchemy import func
+
+def _get_jenkins_aggregation(filters=None, project=None, page=1, page_size=25, run_limit=None):
```

Comment:
This was copy and paste from how the Jenkins Job widget-config works. @rsnyman is this necessary? I can't even tell if this ever gets run. If so it needs to be more personalized for comparison widget cause this is kind of an unknown hack to me.

Reply:
Yeah will do. Jenkins Jobs was the only other full page view I knew about when I started so I thought I needed that to get the widget config up.